docker/run_config.py

```diff
@@ -13,7 +13,7 @@ def main():
 
     args = parser.parse_args()
 
-    flags_parser.parse(args.flags_file, args.remainder)
+    flags_parser.parse(args.flags_file, remainder=args.remainder)
```

Comment:
Can you please comment on this line? It looks like a hyperparameter to the network.

Reply:
Done

Reply:
Resolved.